ts/src/okx.ts

```diff
@@ -841,14 +841,14 @@ export default class okx extends Exchange {
             'commonCurrencies': {
                 // the exchange refers to ERC20 version of Aeternity (AEToken)
                 'AE': 'AET', // https://github.com/ccxt/ccxt/issues/4981
-                'BOX': 'DefiBox',
-                'HOT': 'Hydro Protocol',
+                'BOX': 'DEFIBOX',
```

Comment:
the thing is, this would be a breaking change now

Reply:
but most of the tokens no longer exist in exchange currencies response. only WINTOKEN is still being traded

Reply:
but as you think, if it is not mergeable, please feel free to close PR ✌️

Reply:
so, i was going to do the same to other exchanges, so you say that we shouldn't do that, right? just confirm